src/main/java/calculator/Positives.java

```diff
@@ -0,0 +1,23 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Positives {
+
+    private List<Positive> positiveList;
+
+    public Positives(String[] textArray) {
+        positiveList = new ArrayList<>();
+
+        for(String text : textArray){
+            positiveList.add(new Positive(text));
+        }
+    }
```

Comment:
이번 미션 동안 배열보다 컬렉션에 익숙해 지는 것은 어떨까요? 
다양한 기능을 제공하고 있습니다! 

```suggestion
    public Positives(String[] textArray) {
        positiveList = Arrays.stream(textArray)
            .map(Positive::new)
            .toList();
    }
```